### DISCUSSION TITLE
Telegram Persistent Timestamp Outdated Error and GetChannel Difference Request errors

### DISCUSSION THREAD

**Discussion by ZWECE** [community/community#137099] [Programming Help]
### Select Topic Area

Question

### Body

Hello,

I have many bot versions. My bot runs each account in a loop with joining the group, sending messages and adding reactions to messages.

2 days ago my v1/v7 versions all give the following issue.
My bots that have been running for 3 months without interruption are giving this problem even though I have not made any changes.

What did I do?

Proxy, device, Mac address, phone numbers, wiretapped or unwiretapped session files, different apis, ignoring issues, windows time settings, creating new session files and hundreds of other changes and tests.

1 API key had the power to process my 8 accounts every 10 seconds.

What changed in Telegram and how can I solve the problem?

Thanks for your help.

`Telegram is having internal issues PersistentTimestampOutdatedError: Persistent timestamp outdated (caused by GetChannelDifferenceRequest)`

`Getting difference for channel updates 1645141274 caused ValueError; ending getting difference prematurely until server issues are resolved
`

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @ZWECE ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
